features/org.wso2.carbon.dashboards.migration.tool.feature/src/main/resources/migration.xml

```diff
@@ -0,0 +1,65 @@
+<?xml version="1.0" encoding="ISO-8859-1"?>
+<!-- There are two ways of using the migration script. If you uncomment the CAR file section, you can migrate your car files.
+If you want to migrate the portal app, you can uncomment the Portal File migration section -->
+
+<!-- CAR File Migration-->
+<!--
+	1.Type - Specifiy the migration type CAR or Portal
+
+	2.SourceDir - Specify the car file direcotory or single car file path. If you put directory, it will get all the CAR files in
+	it and migrate it
+
+	3.DestinationDir - Specify the destination directory for migrated car files
+-->
+
+<!-- Uncomment the below part to enable CAR file migration -->
+<DSMigration>
+    <Type>CAR</Type>
+    <SourceDir>/home/nisala/WSO2/TR/CAR/</SourceDir>
+    <DestinationDir>/home/nisala/WSO2/TR/Btest/Modified/</DestinationDir>
+</DSMigration>
+
+        <!-- Portal File Migration-->
+        <!--
+            1.Type - Specifiy the migration type CAR or Portal
```

Comment:
It would be better if you could put these explantations just before the actual configuration below.